.circleci/config.yml

```diff
@@ -244,17 +219,15 @@ commands:
       - store_artifacts:
           path: e2e/logs
           destination: logs
+          when: always
       - store_test_results:
           path: ~/workbench/e2e/logs
 
-  start_local_api:
-    description: "Start local API server"
+  launch-local-api:
```

Comment:
For next time, it would be good to separate the renames from the main PR. It's not worth splitting these out, though.